apps/vanilla/src/pages/black-jack/models/diceGenerator.ts

```diff
@@ -0,0 +1,42 @@
+import { Subscriber } from '../type/subscriber';
+
+import { Publisher } from './publisher';
+const NUMBER_DICE = 6;
+
+/** Generating dice with publisher and subscriber func. */
+export class DiceGenerator extends Publisher<number> implements Subscriber<number> {
```

Comment:
Class documentation should describe purpose of the class not its content.
For instance, JSDoc might look like this:
```suggestion
/** Generator for dice rolls. */
export class DiceGenerator extends Publisher<number> implements Subscriber<number> {
```